app/src/main/java/com/example/marvelapp/api/service/CharacterServiceImpl.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.marvelapp.api.service
+
+import com.example.marvelapp.api.MarvelAppApi
+import com.example.marvelapp.api.mapper.CharacterMapper
+import com.example.marvelapp.api.util.Result
+import com.example.marvelapp.entity.Character
+
+class CharacterServiceImpl : CharacterService {
+
+    private val api = ServiceGenerator()
+    private val mapper = CharacterMapper()
```

Comment:
If you improve this to be an object or an extension function, this will not be necessary

Reply:
done